code-challenges.test.js

```diff
@@ -50,3 +153,35 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+
+//Create a function called 'fullHouse' that takes in an array as a parameter
+//Create a variable called sorted to hold the array with '.sort()' applied
+// Use conditional statement and equality operators
+// Since the array is sorted, use the only 2 possibilities to see if the values at certain indexes give us a fullhouse. (3 of a kind followed by a pair, or a pair followed by 3 of a kind.)
+// Return true if condition is met, else return false
+
+const fullHouse = (array) => {
+    const sorted = array.sort();
+    if (
+      (sorted[0] === sorted[1] &&
+        sorted[1] === sorted[2] &&
+        sorted[3] === sorted[4]) ||
+      (sorted[0] === sorted[1] &&
+        sorted[2] === sorted[3] &&
+        sorted[3] === sorted[4])
+    ) {
+      return true;
+    } else {
+      return false;
+    }
+  };
+  
```

Comment:
Very logical thought process, nice job! Similar thing here, how could you refactor for better efficiency? Just leaving these here if you wanted an additional challenge, not required at all!